beaker-hiera.gemspec

```diff
@@ -30,5 +30,5 @@ Gem::Specification.new do |s|
   s.add_development_dependency 'yard'
 
   # Run time dependencies
-  s.add_runtime_dependency 'beaker', '~> 4.0'
+  s.add_runtime_dependency 'beaker', '~> 5.0'
```

Comment:
Why is it now incompatible with beaker 4? I don't think any API was changed that it depends on so I'd prefer to make it `>= 4 < 6`

Reply:
okay, got that. thx.